with-tinybase/App.js

```diff
@@ -77,6 +77,15 @@ const Todo = ({ rowId }) => {
   );
 };
 
+// A list component to show all the todos.
+const Todos = () => (
+  <FlatList
+    data={useSortedRowIds(TODO_TABLE, DONE_CELL)}
+    renderItem={({ item: id }) => <Todo id={id} />}
```

Comment:
this is like the FlatList documentation but I wonder if the function should be memoized